osu.Game/Screens/Edit/Compose/Components/Timeline/Timeline.cs

```diff
@@ -82,15 +82,16 @@ public Timeline(Drawable userContent)
         private Container mainContent;
 
         private Bindable<float> waveformOpacity;
-        private Bindable<bool> controlPointsVisible;
+        private Bindable<bool> alwaysShowControlPoints;
+        private readonly Bindable<bool> controlPointsVisible = new Bindable<bool>();
         private Bindable<bool> ticksVisible;
 
         private double trackLengthForZoom;
 
         private readonly IBindable<Track> track = new Bindable<Track>();
 
         [BackgroundDependencyLoader]
-        private void load(IBindable<WorkingBeatmap> beatmap, OsuColour colours, OsuConfigManager config)
+        private void load(IBindable<WorkingBeatmap> beatmap, OsuColour colours, OsuConfigManager config, Editor editor)
```

Comment:
`Editor` has to be nullable for some tests to pass, as can be seen from CI failures.